### DISCUSSION TITLE
Conta desativada

### DISCUSSION THREAD

**Discussion by LeoAmbrosioA** [community/community#143799] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### GitHub Feature Area

Actions

### Body

My professor on GitHub, Github.com/joaopauloaramuni, had his account blocked because he was teaching some courses for us and needed to use executable files. Unfortunately, these files were mistakenly flagged as spam, but this was an error. He is very active on the platform, where he regularly uses its resources to teach students.

**Comment by Roland525** [comment, marked as answer]
Go to [GitHub Support](https://support.github.com/).
Select "Contact GitHub Support."
Provide the details about your professor's account, explaining that the executable files were flagged mistakenly.
Mention that the professor is an active educator, and this was an error.

**Reply by queenofcorgis** [reply]
This is the correct method. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community will be unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
